login.js

```diff
@@ -0,0 +1,75 @@
+const bcrypt = require('bcrypt');
+const { app, databaseConnection, con } = require('./app');
+
+
+const init = async () => {
```

Comment:
Why is there a init method here? You can just have a app.post method directly in this file right?